at_hello_world/server_demo_complete/pubspec.yaml

```diff
@@ -4,17 +4,17 @@ publish_to: 'none'
 version: 0.1.0+1
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
   cupertino_icons: ^1.0.0
```

Comment:
Should cupertino_icons be bumped to 1.0.1 to get the minimal Null safety version?

Reply:
Well it didn't show any error.  But I think I should include the latest null safety version of it. Thanks!

Reply:
So it doesn't show any difference if I bumped to 1.0.3. I have checked.

Reply:
What difference would you expect to see?

Reply:
Will Show  errors in the project?